docs/external_system.md

```diff
@@ -241,187 +262,61 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
-
-1. Generate a default SAT configuration file, if one does not exist.
+   The admin account used to authenticate with `sat auth` must be enabled in
+   Keycloak and must have its *assigned role* set to *admin*. For more
+   information on editing *Role Mappings*, see *Create Internal User Accounts
+   in the Keycloak Shasta Realm* in the
+   [Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
+   For more information on authentication types and authentication credentials,
+   see [SAT Command Authentication](introduction.md#sat-command-authentication).
 
    ```screen
-   ncn-m001# sat init
-   Configuration file "/root/.config/sat/sat.toml" generated.
-   ```
-
-   **Note:** If the config file already exists, it will print out an error:
-
-   ```screen
-   ERROR: Configuration file "/root/.config/sat/sat.toml" already exists.
-   Not generating configuration file.
-   ```
-
-1. Edit `~/.config/sat/sat.toml` and set the username option in the `api_gateway` section of the config file. For
-   example:
-
-   ```screen
-   username = "crayadmin"
-   ```
-
-1. Run `sat auth`. Enter your password when prompted. For example:
-
-   ```screen
-   ncn-m001# sat auth
+   $ sat auth
    Password for crayadmin:
    Succeeded!
    ```
 
-1. Other `sat` commands are now authenticated to make requests to the API gateway. For example:
-
-   ```screen
-   ncn-m001# sat status
-   ```
-
-## Generate SAT S3 Credentials
-
-Generate S3 credentials and write them to a local file so the SAT user can access S3 storage. In order to use the SAT
-S3 bucket, the System Administrator must generate the S3 access key and secret keys and write them to a local file.
-This must be done on every Kubernetes master node where SAT commands are run.
-
-SAT uses S3 storage for several purposes, most importantly to store the site-specific information set with `sat setrev`.
-
-### Prerequisites
-
-- The SAT CLI has been installed following [Install and Configure SAT](#install-and-configure-sat).
-- The SAT configuration file has been created following [Authenticate SAT Commands](#authenticate-sat-commands).
-- CSM has been installed and verified.
-
-### Procedure
-
 1. Ensure the files are readable only by `root`.
 
     ```screen
-    ncn-m001# touch /root/.config/sat/s3_access_key \
-        /root/.config/sat/s3_secret_key
+    $ touch ~/.config/sat/s3_access_key \
+        ~/.config/sat/s3_secret_key
     ```
 
     ```screen
-    ncn-m001# chmod 600 /root/.config/sat/s3_access_key \
-        /root/.config/sat/s3_secret_key
+    $ chmod 600 ~/.config/sat/s3_access_key \
+        ~/.config/sat/s3_secret_key
     ```
 
 1. Write the credentials to local files using `kubectl`.
 
+  Generate S3 credentials and write them to a local file so the SAT user can
+  access S3 storage. In order to use the SAT S3 bucket, the system
+  administrator must generate the S3 access key and secret keys and write them
```

Comment:
Since this is happening on an external system, "user" is more appropriate than "system administrator".

I also think "generate" is a little misleading. This step really just gets these values from a Kubernetes secret and stores them in files. I know that "generate s3 credentials" terminology is used elsewhere, so feel free to defer that piece of feedback if you prefer.

```suggestion
  access S3 storage. In order to use the SAT S3 bucket, the user
  must obtain the S3 access key and secret keys and write them
```

Reply:
Updated in next commit.